advanced_functionality/autogluon-sagemaker-pipeline/README.md

```diff
@@ -0,0 +1,110 @@
+# Autogluon Tabular with SageMaker Pipeline
+This example shows how to use [AutoGluon](https://github.com/awslabs/autogluon) with SageMaker Pipelines. To create a SageMaker pipeline, please follow the steps in tutorials such as [this](https://aws.amazon.com/sagemaker/pipelines/). After the pipeline creation is complete, come back here to do modifications for Autogluon. You need to add below files to the official repo:
```

Comment:
For accessibility, please use the full title of linked resources for link text instead of "this", such as "Amazon SageMaker Pipelines" / "this Amazon SageMaker Pipelines tutorial".

Reply:
addressed this comment